packages/micro-analytics-cli/src/handler.js

```diff
@@ -24,6 +24,13 @@ function realtimeHandler(req, res) {
   }
 }
 
+async function readMeta(req) {
+  if (await text(req)) {
+    return (await json(req)).meta;
```

Comment:
Why do we want to crash the server if some random person submits bad JSON? That seems pretty bad?

```JS
try {
  const parsed = await json(req);
  return parsed || null;
} catch (err) {
  console.log('bad data')
  console.log(err);
}
```